### DISCUSSION TITLE
Error 02-JS.I homework

### DISCUSSION THREAD

**Discussion by jepon26** [atralice/Curso.Prep.Henry#138] [General]
Buen dia

Me sale un error 

esVocal(letra) › Deberia devolver el string "Es vocal", si el valor ingresado lo es

    expect(received).toBe(expected) // Object.is equality

    Expected: "Es vocal"
    Received: "Dato incorrecto"

hice o siguiente:

if(letra.length > 1){
    return "Dato incorrecto"
  }
  if(letra === "A" || letra === "E" || letra === "i" || letra === "o" || letra === "U" ){
    return "Es vocal"
  }
  return "Dato incorrecto"

}

**Comment by Shamyxconsh** [comment]
Hola!! me salta el mismo error, alguien pudo solucionarlo?

**Reply by jepon26** [reply]
Prueba este, asi me funciono:

 return "Es vocal"
  }
  return "Dato incorrecto"

**Reply by Shamyxconsh** [reply]
Gracias!!!!!

**Reply by eze-alvarez** [reply]
faltaba colocar el ; al terminar cada return

**Comment by nicopalomares** [comment]
hola disculpen les hago una consulta,. Estoy realizando la tarea 02-js-I y mientras las realizo cuando pongo para hacer el test me sale como si no se haya modificado nada,tambien pongo "control s" para guardar pero de todas formas en el test me sale como si no se hubiera hecho ningun cambio

**Reply by jepon26** [reply]
Puede ser que no esté direccionado a la carpeta 2 del homework, talves viendo la imagen

**Comment by Shamyxconsh** [comment]
Hola!!! Me pasa exactamente lo mismo!!! Estoy demorada por eso! Quiero
ejecutar para hacer las correcciones y me tira que no hay nada
